src/oracles/UniswapV2Oracle.sol

```diff
@@ -42,11 +43,12 @@ contract UniswapV2Oracle {
         price0CumulativeLast = pair.price0CumulativeLast();
         price1CumulativeLast = pair.price1CumulativeLast();
         
-        price0Average = FixedPoint.uq112x112(uint224(pair.price0CumulativeLast()));
-        price1Average = FixedPoint.uq112x112(uint224(pair.price1CumulativeLast()));
+        (uint112 reserve0,,) = pair.getReserves();
+        (,uint112 reserve1,) = pair.getReserves();
```

Comment:
```suggestion
        (uint112 reserve0, uint112 reserve1, ) = pair.getReserves();
```

Just a nit, but destructuring assignment tends to have spaces to make it more readable. Check out the example here: https://docs.soliditylang.org/en/v0.8.11/control-structures.html#destructuring-assignments-and-returning-multiple-values

Also, you can just destructure and assign in one line, no?